impeller/compiler/reflector.cc

```diff
@@ -624,6 +620,17 @@ std::vector<size_t> Reflector::ComputeOffsets(
   return offsets;
 }
 
+std::optional<size_t> Reflector::GetOffset(
+    spirv_cross::ID id,
+    const std::vector<size_t>& offsets) const {
+  uint32_t location =
```

Comment:
I didn't know there was just a tool at the time impellerc was written, but there is spriv-val that we can run over the intermediate Stage 1 SPIRV to avoid getting into this situation. But being defensive here make sense to me.